### PR TITLE
Chapter 3 practic

### PR DESCRIPTION
**Pull request by Dmitriy-Romanchuk**

### CODE REVIEW

File: chapter_3_practic/task11.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		int  pounds = 0, shillings = 0, pence = 0; // for results
+		float multiplay = 0.0f;
+		char ch = ' ';
+
+		std::cout << "Enter first amount: (5.10.6)"; // Запрос ввода пользователем первого значения
+		std::cin >> pounds1 >> ch >> shillings1 >> ch >> pence1;
+
+		std::cout << "Enter operation '+', '-' or '*'";// Выбор операции: сложение, вычитание, умножение и запрос на ввод значений
+		std::cin >> sign;
+		if (sign != '*')
```
Comment (zefir-o) [inline chapter_3_practic/task11.cpp]: if you restructure the code like
if ('*' == sign)
{
// do for multiplier
}
it's easier to read in comparison to when you have != operator
Also, prefer the style 
if (/*constant*/ value == changeable)
in this case means if ('*' == value)
such style prevents the case if (value = '*') which you could make by mitake

File: chapter_3_practic/task12.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+			break;
+			default:
+				std::cout << "Warning: \"" << sign << "\" is unknown operator" << std::endl;
+		}
+
+			std::cout << "Do another(y / n) ? "; // воспрос повторить или нет
+			std::cin >> repeat;
+		}
+
+		return EXIT_SUCCESS;
```
Comment (zefir-o) [inline chapter_3_practic/task12.cpp]: wrong formatting

**Review by zefir-o — APPROVED** [review]

File: chapter_3_practic/task11.cpp (hunk 1 above)
Comment (Dmitriy-Romanchuk) [reply chapter_3_practic/task11.cpp]: https://github.com/Dmitriy-Romanchuk/master/pull/3/commits/90cb98bbed2eadee4be5c4c4c8b743da3a1e4554

File: chapter_3_practic/task12.cpp (hunk 2 above)
Comment (Dmitriy-Romanchuk) [reply chapter_3_practic/task12.cpp]: https://github.com/Dmitriy-Romanchuk/master/pull/3/commits/90cb98bbed2eadee4be5c4c4c8b743da3a1e4554

**Pull request merged**